python/pyarrow/_dataset.pyx

```diff
@@ -2484,9 +2482,11 @@ cdef class Scanner(_Weakrefable):
             record batches are overflowing memory then this method can be
             called to reduce their size.
         batch_readahead : int, default 16
-            The number of batches to read ahead in a file. This might not work
-            for all file formats. Increasing this number will increase
```

Comment:
Same here

Reply:
Good catch. When I was in there fixing this, I found one more instance that wasn't consistent. All three instances of this docstring now have the same text,

> The number of batches to read ahead in a file. This might not work
        for all file formats. Increasing this number will increase
        RAM usage but could also improve IO utilization.